lib/iterators.js

```diff
@@ -125,6 +125,11 @@ function traverseAsyncApiDocument(doc, callback, schemaTypesToIterate) {
     Object.values(doc.components().schemas()).forEach(s => {
       traverseSchema(s, callback, null, schemaTypesToIterate);
     });
+    if (doc.components().hasMessages()) {
```

Comment:
It's a strange 😄 For schemas we didn't check that schemas exist in components, but for messages we'll have this statement 😄

Reply:
Yea, I was actually getting failed tests without this statement 😕

Reply:
Moved some checks around so the function traverseMessage now checks for null messages instead :)